joplin/api/schema.py

```diff
@@ -6,19 +6,116 @@
 from graphene_django.filter import DjangoFilterConnectionField
 from graphene.types import Scalar
 from graphene.types.generic import GenericScalar
-from wagtail.core.fields import StreamField
+from wagtail.core.fields import StreamField, RichTextField
 from wagtail.core.models import Page, PageRevision
 from django_filters import FilterSet, OrderingFilter
-from wagtail.core.blocks import PageChooserBlock, TextBlock, ListBlock
+from wagtail.core.blocks import *
 from wagtail.documents.models import Document
-
+from wagtail.core.rich_text import expand_db_html
 from base.models import TranslatedImage, ThreeOneOne, ServicePage, ServicePageContact, ServicePageTopic, ServicePageRelatedDepartments, InformationPageRelatedDepartments, ProcessPage, ProcessPageStep, ProcessPageContact, ProcessPageTopic, InformationPage, InformationPageContact, InformationPageTopic, DepartmentPage, DepartmentPageContact, DepartmentPageDirector, Theme, TopicCollectionPage, TopicPage, Contact, Location, ContactDayAndDuration, Department, DepartmentContact, TopicPageTopicCollection, OfficialDocumentPage, OfficialDocumentPageRelatedDepartments, OfficialDocumentPageTopic, OfficialDocumentPageOfficialDocument, GuidePage, GuidePageTopic, GuidePageRelatedDepartments, GuidePageContact, JanisBasePage, PhoneNumber, DepartmentPageTopPage, DepartmentPageRelatedPage
+import collections
+import traceback
+from nested_lookup import nested_lookup, nested_alter, nested_update, get_occurrence_of_key, get_occurrence_of_value
+
+
+class RichTextFieldType(Scalar):
+    """Serialises RichText content into fully baked HTML
+    see https://github.com/wagtail/wagtail/issues/2695#issuecomment-373002412 """
```

Comment:
went in a loop because this is copied from https://github.com/wagtail/wagtail/issues/2695#issuecomment-464462575

Reply:
I left it in, I still think its a relevant issue to note. I could change it so its just the last comment on that issue though?

Reply:
that would be perfect.